server/src/main/scala/io/delta/sharing/server/model.scala

```diff
@@ -64,6 +67,12 @@ case class Metadata(
   override def wrap: SingleAction = SingleAction(metaData = this)
 }
 
+case class QueryStatus(
+      queryId: String = null
+      ) extends  Action {
```

Comment:
nit: the lint seems weird.

Reply:
Also, doesn't queryStatus contain any status? why it only contains id?

Reply:
currently, we throw exception when there is error happens. and return result if query is success, so getting query status means it's pending, there was a status but I removed after Abhijit's comments, which I think it makes sense.